pkg/ucglib/src/csrc/ucg_riotos.c

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (C) 2018 Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
```

Comment:
missing doxygen header, though its not as import for C files as for header files

Reply:
Added header.